applications/luci-app-smartdns/root/etc/uci-defaults/50_luci-smartdns

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
Your OpenWrt init script for smartdns specifies `procd_add_reload_trigger smartdns` so you can drop the entire ucitrack integration. The index cache removal has been made generic in https://github.com/openwrt/luci/commit/1f8a3c04178e21f45f3908fd476ec3b3dad73c69 so you can drop the whole uci-defaults file.